### DISCUSSION TITLE
How to generate a cylinder and gridding?

### DISCUSSION THREAD

**Discussion by lipxisi** [idaholab/moose#23773] [Q&A Meshing] [answered]
I want to generate a simple cylinder using mesh generators, but haven't find the specialized generator. 
Which generator should I use and  what`s the code?
Thanks for your answer!

**Comment by GiudGiud** [comment, marked as answer]
Hello

You ll want to use this one
https://mooseframework.inl.gov/docs/site/source/meshgenerators/ConcentricCircleMeshGenerator.html
then this one
https://mooseframework.inl.gov/docs/site/source/meshgenerators/MeshExtruderGenerator.html

Guillaume
